src/Support/Tag.php

```diff
@@ -5,36 +5,55 @@
 use Illuminate\Contracts\Support\Renderable;
 use Illuminate\Contracts\View\View;
 use Illuminate\Support\Collection;
+use Illuminate\Support\HtmlString;
 
+/**
+ * A representation of a HTML tag
+ */
 abstract class Tag implements Renderable
 {
-    protected static array $reservedAttributes = [
-        'tag',
-        'inner',
-        'attributesPipeline',
-    ];
+    const ATTRIBUTES_ORDER = ['rel', 'hreflang', 'title', 'name', 'href', 'property', 'description', 'content'];
 
+    /**
+     * The HTML tag
+     */
     public string $tag;
 
+    /**
+     * The HTML attributes of the tag
+     */
+    public array $attributes = [];
+
+    /**
+     * The content of the tag
+     */
+    public null|string|HtmlString $inner = null;
+
     public array $attributesPipeline = [];
 
     public function render(): View
     {
         return view('seo::tags.tag', [
             'tag' => $this->tag,
             'attributes' => $this->collectAttributes(),
-            'inner' => $this->inner ?? null,
+            'inner' => $this->inner,
         ]);
     }
 
     public function collectAttributes(): Collection
     {
-        return collect($this->attributes ?? get_object_vars($this))
```

Comment:
I'm removing `get_object_vars($this)` as it can could produce unexpected conflicts and create confusion on how things are generated. The `$attributes` property is now the only way to go

Reply:
Could you elaborate a bit on the reasoning behind this? More a preference or functional?

If you ask me this particular aspect doesn't need to be refactored, because it worked fine so far. The idea behind these classes that extend `Tag` is that each class can be considered as representing a tag as `<tag></tag>`, with the top-level properties on the class representing the `<tag property="value"></tag>`. So I would prefer to put these few properties back and have the `get_object_vars()` as option as well.

Reply:
I do see the benefits of having a "pure" representation of the tag using `get_object_vars`, what I don't understand is the fact that the class also has a `$attributes` property. From my point of view, this:
`$this->attributes ?? get_object_vars($this)` 
was very confusing because it introduces two paradigms in the same representation, one overriding the other.

The `Schema` class for example was confusing because it was defined with the following properties:
```php
abstract class Schema extends Tag
{
    use Pipeable;

    public array $attributes = [
        'type' => 'application/ld+json',
    ];

    public string $context = 'https://schema.org/';
```
`$context` was actually never used because of `$attributes`. 

I do believe the package should have only one way to define `$attributes`, and I picked the `$attributes` property because it is safer and clearer than `get_object_vars`. 
The later option would put you at risk to collide with the already reserved attributes `$tag``, $inner` and `$attributes` and you would not be able to define any property without having it rendered as a tag attributes. 

I guess that's why you introduced the second paradigm `$attributes` in the first place (Schema classes have a lot of properties that are not tag attributes)